docs/sources/features/datasources/cloudwatch.md

```diff
@@ -114,20 +109,62 @@ aws_secret_access_key = dasdasdsadasdasdasdsa
 region = us-west-2
 ```
 
-## Metric Query Editor
+## Using the M  etric Query Editor
+
+You need to specify a namespace, metric, at least one stat, and at least one dimension as long as `Match Exact` is activated. In the case `Match Exact` is not activated, no dimension needs to be specified.
+
+### Dynamic queries using dimension wildcards
+
+In Grafana 6.5 or higher, you’re able to monitor a dynamic list of metrics by using the asterisk (\*) wildcard for one or more dimension values.
+
+{{< docs-imagebox img="/img/docs/v65/cloudwatch-dimension-wildcard.png" max-width="800px" class="docs-image--right" caption="CloudWatch dimension wildcard" >}}
+
+In the example, all metrics in the namespace `AWS/EC2` with a metric name of `CPUUtilization` and ANY value for the `InstanceId` dimension are queried. This can help you monitor metrics for AWS resources, like EC2 instances or containers. For example, when new instances get created as part of an auto scaling event, they will automatically appear in the graph without you having to track the new instance IDs. This capability is currently limited to retrieving up to 100 metrics. You can click on `Show Query Preview` to see the search expression that is automatically built to support wildcards. To learn more about search expressions, visit the [CloudWatch documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/search-expression-syntax.html).
 
-![](/img/docs/v43/cloudwatch_editor.png)
+By default, the search expression is defined in such a way that the queried metrics must match the defined dimension names exactly. This means that in the example below only metrics with exactly one dimension with name ‘InstanceId’ will be returned.
 
-You need to specify a namespace, metric, at least one stat, and at least one dimension.
+You can untoggle `Match Exact` to include metrics that have other dimensions defined. Disabling ‘Match Exact’ also creates a search expression even if you don’t use wildcards. We simply search for any metric that match at least the namespace, metric name, and all defined dimensions.
 
-## Metric Math
+### Multi-value template variables now use search expressions
+
+When defining dimension values based on multi-valued template variables,a search expressions is used to query for the matching metrics. This enables the use of multiple template variables in one query and also allows you to use template variables for queries that have the `Match Exact` option disabled.
```

Comment:
- Need a space after `template variables,`
- Should be `a search expression` (no s)